src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2570,7 +2570,6 @@ because xbuild doesn't support framework reference assemblies.
     ManifestFile="$(_ProjectAndroidManifest)"
     BuildToolsVersion="$(AndroidSdkBuildToolsVersion)"
     PlatformToolsVersion="$(AndroidSdkPlatformToolsVersion)"
-    ToolsVersion="$(AndroidSdkToolsVersion)"
```

Comment:
Can we delete the property on the task, too?

https://github.com/xamarin/xamarin-android/blob/dfed2f2db0216e0e5a16224faaee7f868752938b/src/Xamarin.Android.Build.Tasks/Tasks/CalculateProjectDependencies.cs#L32

Reply:
Done.